modules/nf-neuro/connectivity/afdfixel/main.nf

```diff
@@ -0,0 +1,49 @@
+process CONNECTIVITY_AFDFIXEL {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ 'scilus/scilus:latest' }"
+
+    input:
+    tuple val(meta), path(h5), path(fodf)
```

Comment:
There is no collision between input and output, so you can uniformize the names to both `hdf5`